### DISCUSSION TITLE
Feedback: 1.x → 2.x

### DISCUSSION THREAD

**Discussion by v-mokrushin** [mswjs/mswjs.io#375] [Q&A]
- [Open page](https://mswjs.io/docs/migrations/1.x-to-2.x/)

[Request/Response/TextEncoder is not defined (Jest)](https://mswjs.io/docs/migrations/1.x-to-2.x/#requestresponsetextencoder-is-not-defined-jest)

The solution does not work. Still the same error.

**Comment by kettanaito** [comment]
Hi, @v-mokrushin. I'm sorry you're experiencing trouble getting this to work. 

Please take a look at our [Jest/JSDOM + MSW](https://github.com/mswjs/examples/tree/main/examples/with-jest-jsdom) example that features the entire setup. It can help you spot what's missing.
